packages/task/src/browser/task-configurations.ts

```diff
@@ -317,7 +317,7 @@ export class TaskConfigurations implements Disposable {
 
         const configuredAndCustomizedTasks = await this.getTasks(token);
         if (!configuredAndCustomizedTasks.some(t => this.taskDefinitionRegistry.compareTasks(t, task))) {
-            await this.saveTask(scope, { ...task, problemMatcher: [] });
+            await this.saveTask(scope, task);
```

Comment:
Is there any risk associated with accepting any problem matchers coming into this call? Was the `problemMatcher: []` just always a bad idea?

Reply:
I don't see any risks with it, it just seem to have been left as an initial default. 
Please let me know if you see any issues with it.